EventListener/SecurityListener.php

```diff
@@ -48,7 +54,7 @@ public function onKernelController(FilterControllerEvent $event)
             return;
         }
 
-        if (null === $this->securityContext || null === $this->trustResolver) {
+        if (null === $this->tokenStorage || null === $this->trustResolver) {
```

Comment:
Should 
```php 
null === $this->tokenStorage
``` 
be something like
```php
(null === $this->securityContext && null === $this->tokenStorage)
```
?

```$tokenStorage``` will always be null for symfony prior 2.6, i think